<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>courier_company_window</class>
 <widget class="QWidget" name="courier_company_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QListWidget" name="packagesListWidget"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="recipientLabel">
         <property name="text">
          <string>Recipient:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="recipientLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="streetLabel">
         <property name="text">
          <string>Street:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="streetLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="streetNrLabel">
         <property name="text">
          <string>Street number:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="streetNrLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="xLocLabel">
         <property name="text">
          <string>X Location:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="xLocLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="yLocLabel">
         <property name="text">
          <string>Y Location:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="yLocLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
